Lab_1b/Syntax.in

```diff
@@ -0,0 +1,43 @@
+nr = "0" | nzd{digit}
+nzd = "1" | "2" | ... | "9"
+digit = "0" | nzd
+
+program = "{" {declaration} stmtlist "}"
+
+declaration = identifier ":" type ";"
+
+type = primitive | arraydeclr
+
+primitive = "int" | "char" | "bool" | "string" 
+
+arraydeclr = "array" "[" nr "]" primitive
+
+stmtlist = stmt ";" {stmt ";"}
+
+stmt = simplStmt | cmpStmt | ifStmt | whileStmt
+
+simplStmt = assignStmt | ioStmt
+
+assignStmt = identifier "=" expr ";"
+
+expr = term {operator term}
```

Comment:
you cannot have "()" in expressions? What happens in the case when you want to have a complex mathematical operation?